src/v/redpanda/admin/api-doc/debug.json

```diff
@@ -641,9 +638,42 @@
         }
     ],
     "models": {
+        "cpu_profile_result": {
+            "id": "cpu_profile_result",
+            "description": "top level object for a cpu profile request",
```

Comment:
```suggestion
            "description": "Top-level object for a CPU profile request",
```
Style consistency with the rest of spec and public docs

Reply:
@kbatuigas thanks for all these. I pained me to write in "texting" style, but the nearby descriptions for this type all used the same style, but looking at the wider file I see it's actually mostly just the cpu profiler, the rest of the stuff does use Caps + no period style, so thanks for calling this out!

I'll also fix the other descriptions in cpu_profiler in a separate commit.